src/bootlint.js

```diff
@@ -938,6 +938,16 @@ var LocationIndex = _location.LocationIndex;
             reporter('`.form-control` cannot be used on non-textual `<input>`s, such as those whose `type` is: `file`, `checkbox`, `radio`, `range`, `button`', formControlsOnWrongTypes);
         }
     });
+    addLinter("E043", function lintNavbarButtons($, reporter) {
+        $('.navbar-nav').find('a').each(function () {
```

Comment:
I would refactor this to let the browser do the heavy lifting, to something like:
```js
var navAnchorBtns = $('.navbar-nav a.btn');
...
var anchorNavbarBtns = $('.navbar-nav a.navbar-btn');
...
```